src/index.js

```diff
@@ -22,6 +23,8 @@ export default function supernova(env) {
       const [pageInfo, setPageInfo] = useState({ top: 0, height: 100 });
       const [tableData, setTableData] = useState();
 
+      const selections = initSelections();
+
```

Comment:
Can you make selections dependant on options? So that it can be disabled from the getgo. Both for edit mode cases but also snapshots and directly from a mashup emebd.
Think it is useOptions?

Reply:
that should work